student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
```

Comment:
- [ ] Please consider using `require_relative` when you're importing files in your own project. It's a recommended practice to keep `require` for external libraries/gems! 🤓 🔧